app/views/dishes/index.html.haml

```diff
@@ -0,0 +1,31 @@
+.row
+  .small-3.small-centered.columns
+
+  %h1 Dishes
+
+  %table
+    %thead
+      %tr
+        %th Name
+        %th Alergy Information
+        %th Price
+        %th Calories
+        %th Image
+        %th
+        %th
+        %th
+
+    %tbody
+      - @dishes.each do |dish|
+        %tr
+          %td= link_to dish.name, dish
```

Comment:
Is this link actually pointing to the dish show page?  I would think that the markup should be:
```
link_to dish.name, dish_path(dish)
```